tests/test_data.py

```diff
@@ -422,33 +425,6 @@ async def assert_setting_in_db(expected_autostart_agent_map: dict[str, object])
     assert assert_setting_in_db(autostart_agent_map)
 
 
-async def test_environment_deprecated_setting(init_dataclasses_and_load_schema, caplog):
```

Comment:
Why was this test case removed?

Reply:
This was a mistake, I added it back. I'll leave it up to you to merge this PR :) Thanks for everything